lib/oli_web/components/delivery/practice_activities/practice_activities.ex

```diff
@@ -575,7 +575,7 @@ defmodule OliWeb.Components.Delivery.PracticeActivities do
          %Section{analytics_version: :v2, id: section_id},
          student_ids
        ) do
-    page_type_id = Oli.Resources.ResourceType.id_for_activity()
+    page_type_id = Oli.Resources.ResourceType.get_id_by_type("activity")
```

Comment:
Maybe we could add an alias for this module and call that alias here (call `ResourceType` only)